web/react/stores/user_store.jsx

```diff
@@ -64,12 +65,12 @@ var UserStore = assign({}, EventEmitter.prototype, {
   },
   setCurrentId: function(id) {
     if (id == null)
-      sessionStorage.removeItem("current_user_id");
+      localStorage.removeItem("current_user_id");
```

Comment:
Is this suppose to be BrowserStore?  also next couple of lines refer to localStorage too.